Cargo.toml

```diff
@@ -63,6 +64,11 @@ accesskit = "0.19.0"
 accesskit_winit = "0.27.0"
 accesskit_consumer = "0.28.0"
 time = "0.3.41"
+reqwest = { version = "0.12.20", default-features = false, features = [
+    # We use rustls as Android doesn't ship with openssl
+    # and this is likely to be easiest to get working.
+    "rustls-tls",
```

Comment:
Probably for backwards compatibility(?) but I think this should be the default at this point anyway doesn't it?

Reply:
> but I think this should be the default at this point anyway doesn't it?

FWIW, I've run into quite a lot of websites that don't work with rustls (because they target an older version of TLS/SSL). So that probably depends on whether you have control over the servers you need to connect to or whether you accept arbitrary URLs.l

Reply:
Do you want us to remove this comment?

Reply:
Comment seems good to me. Probably not necessary, but nice to know for anyone reading the `Cargo.toml`.